about.html

```diff
@@ -0,0 +1,190 @@
+              <stop offset="0.315625" stop-color="#00057D" />
+              <stop offset="0.609254" stop-color="#385E46" stop-opacity="0.4602" />
+              <stop offset="0.733513" stop-color="#04813D" stop-opacity="0.7349" />
+              <stop offset="0.893906" stop-color="#C0BA1B" stop-opacity="0.7349" />
+            </linearGradient>
+          </defs>
+        </svg></a>
+      <div class="collapse navbar-collapse justify-content-lg-start" id="navbarNav">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link mx-5 fs-4" href="index.html">Accueil</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active mx-5 fs-4" href="about.html">À Propos</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link mx-5 fs-4" href="my_projects.html">Mes Projets</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link mx-5 fs-4" href="Contact.html">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <!-- Main -->
+
+  <main class="container-fluid">
+    <section class="text-white py-5">
+      <div class="container text-center">
+        <img src="images/ma_photo.jpg" alt="Ma Photo Personnel" class="rounded-circle my-3 img-fluid" width="200">
+        <h1>Bienvenue</h1>
+        <p class="lead fs-2 lead">Bonjour, je suis un étudiant en développement web passionné par la technologie et la
+          création de solutions innovantes.
+          Je suis actuellement en train de suivre une formation en développement web
+          et je suis à la recherche d'opportunités pour approfondir mes connaissances et acquérir de l'expérience
+          pratique.
+          Je possède une solide connaissance des langages de programmation tels que HTML, CSS, JavaScript, Node.js.
+          J'ai également une expérience dans la création de sites web responsives et de bases de données.
+          Je suis curieux, créatif et passionné par l'apprentissage continu.
+          Je suis convaincu que la technologie peut être utilisée pour résoudre de nombreux problèmes et je suis motivé
+          pour créer des solutions qui auront un impact positif sur la vie des gens.
+          Je suis à la recherche d'opportunités de stage ou d'emploi dans le domaine du développement web, où je pourrai
+          apporter mes compétences et en acquérir de nouvelles.
+          Si vous êtes à la recherche d'un développeur web passionné et talentueux, n'hésitez pas à me contacter.</p>
+
+      </div>
+      <div class="text-lg-center">
+        <img src="images/Curriculum vitae.jpg" alt="Curriculum vitae">
+      </div>
+
+    </section>
+
+  </main>
+
+
+  <!-- footer -->
+  <footer class="text-white my-5 rounded-2">
+
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-3 order-1">
+          <h5>Follow Us</h5>
+
+          <a href="#" class="icon-link icon-link-hover text-light "
```

Comment:
Use only the social networks that you have an account: Linkedin and GitHub. And don't forget the links. Also, take a look in a previous comment that I did about the target property.